AmplifyPlugins/Predictions/AWSPredictionsPlugin/AWSPredictionsPlugin+ClientBehavior.swift

```diff
@@ -46,6 +46,32 @@ extension AWSPredictionsPlugin {
 
     }
 
+    public func convert(speechToText: URL,
+                        options: PredictionsSpeechToTextRequest.Options?,
+                        listener: PredictionsSpeechToTextOperation.EventListener?) -> PredictionsSpeechToTextOperation {
+        let request = PredictionsSpeechToTextRequest(speechToText: speechToText,
+                                                     options: options ?? PredictionsSpeechToTextRequest.Options())
+
+        let multiService = TranscribeMultiService(coreMLService: coreMLService, predictionsService: predictionsService)
+
+        // only one transcription request can be sent at a time
+        if !queue.operations.contains(where: { $0 is AWSTranscribeOperation}) {
+            let operation = AWSTranscribeOperation(request: request,
+                                                   multiService: multiService,
+                                                   requestInProcess: false,
+                                                   listener: listener)
+            queue.addOperation(operation)
+            return operation
+        }
+        else {
+            let operation = AWSTranscribeOperation(request: request,
+                                                   multiService: multiService,
+                                                   requestInProcess: true,
+                                                   listener: listener)
+            return operation
```

Comment:
We need to add the operation to the queue to make it run right?

Also, will it be better to send a new operation object that returns nil always. Something like this:

```
public class AWSTranscribeInvalidOperation: AmplifyOperation<PredictionsSpeechToTextRequest,
    Void,
    SpeechToTextResult,
    PredictionsError>,
PredictionsSpeechToTextOperation {


    init(request: PredictionsSpeechToTextRequest,
         listener: EventListener?) {
        super.init(categoryType: .predictions,
                   eventName: HubPayload.EventName.Predictions.speechToText,
                   request: request,
                   listener: listener)
    }

    override public func main() {

        if isCancelled {
             finish()
             return
         }

        let error = PredictionsError.network(..)
        dispatch(event: .failed(error))
        finish()
        return
    }
}

```